src/components/app/App.js

```diff
@@ -219,7 +223,22 @@ render() {
         }
     ];
 
+
+
     return (
-        <UserDetails />
-    )
+    users.map((el,index) => {
```

Comment:
there has to be space between coma and next word. 
Also, please use meaningful variable names. what is `el`? Should we call it `user`?